config/locales/en.yml

```
@@ -1035,7 +1045,10 @@ en:
         more the delay before time slot opening is important (0 to disallow the
         waiting time).
       optimization_force_start_help: Force vehicle start to the beginning of its schedule.
+      enable_optimization_soft_upper_bound_help: Enable delays for stops and vehicles
```

```suggestion
      enable_optimization_soft_upper_bound_help: Enable the delays in optimization
```
Moreover put it right after `enable_global_optimization_test`